openslides/motions/signals.py

```diff
@@ -68,7 +68,7 @@ def setup_motion_config(sender, **kwargs):
         subgroup=ugettext_lazy('General'))
 
     # Amendments
-
+    # Amendments currently not implemented. (TODO: Implement it like in OpenSlides 1.7.)
```

Comment:
How we can disable the configs that they are defined but are not visible? Currently we can't just remove it because several tests and motion model required these options.